docs/src/main/asciidoc/Getting_Started_Guide.adoc

```diff
@@ -115,6 +115,23 @@ This document provides a quick overview on how to download and get
 started using WildFly 10 for your application development. For in-depth
 content on administrative features, refer to the WildFly 10 Admin Guide.
 
+[[ee8-preview]]
+=== EE8 Preview ===
+By default, WildFly supports Java EE7. We aim to support Java EE8 in the
+future. As we implement Java EE8, and the technologies become available, we offer a preview of these.
+To turn on the Java EE8 preview for these technologies, start the server setting the
+`ee8.preview.mode` system property to `true`. This will switch the technologies
```

Comment:
Does the elytron+JSON-P issue mean that setting this via standalone.xml doesn't really work and that users should set this property from the command line? If so I think the mechanisms that work should be explicitly stated.  (I'm pretty sure the -P switch won't work either as it is processed after JBoss Modules processes the kernel module graph.)